src/engine/controls/ratecontrol.cpp

```diff
@@ -603,3 +616,18 @@ bool RateControl::isReverseButtonPressed() {
     }
     return false;
 }
+
+void RateControl::notifyWrapAround(mixxx::audio::FramePos triggerPos,
+        mixxx::audio::FramePos targetPos) {
+    // TODO FramePos only for easy validation.
+    // Check how to get kLegacyInvalidEnginePosition in order to use sample pos
+    // for the entire
```

Comment:
Why do you want to switch from frame position to sample position?

Reply:
it's passed only to PositionScratchController and uses only sample positions for processing anyway, so I thought we'd better convert it early. No issue though, and getting kLegacyInvalidEnginePosition here requires more includes, so I can as well remove the TODO.